pubsub_nats_test.go

```diff
@@ -30,7 +41,12 @@ func TestNats_NewPubSubServer(t *testing.T) {
 			So(ps.clientID, ShouldNotBeEmpty)
 			So(ps.username, ShouldEqual, "")
 			So(ps.password, ShouldEqual, "")
+			So(ps.retryInterval, ShouldBeGreaterThan, 0)
 			So(ps.tlsConfig, ShouldEqual, nil)
+			// verify that client id is a proper V4 UUID
+			id, err := uuid.FromString(ps.clientID)
+			So(err, ShouldBeNil)
+			So(id.Version(), ShouldEqual, uuid.V4)
```

Comment:
👀 this will prevent someone in the future changing behaviour by not using a UUID for the client ID (that is autogenerated).